### DISCUSSION TITLE
Sign in is broken.

### DISCUSSION THREAD

**Discussion by koaning** [fief-dev/fief#64] [General]
I didn't know where else to notify folks, but the sign-in for fief seems broken. 

<img width="313" alt="CleanShot 2022-08-31 at 21 02 36@2x" src="https://user-images.githubusercontent.com/1019791/187761357-1d6e6422-24bd-49aa-9240-9b10a973d322.png">

**Comment by frankie567** [comment]
Hello @koaning 👋

Thank you for your report! Could you try again? We experience some sporadic DB connection reset which could cause the issue.

**Reply by koaning** [reply]
Yep! Seems all green now. 

Will try to give the service another spin tomorrow. It might be *just* the thing I need for https://calmcode.io and https://dearme.email.

**Reply by koaning** [reply]
No promises though as I'm currently on parental leave. 

so. many. diapers.

**Reply by frankie567** [reply]
Would love to see this!

Good luck with your baby 😅 Enjoy 🍼
